sm/json_sm/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "json_sm"
+version = "0.1.0"
```

I suppose the version should  be align with the rest of tedge.
```suggestion
version = "0.2.3"
```

---

Fixed